### PR TITLE
Use single source of the nessie version

### PR DESCRIPTION
**Pull request by wendigo**
This way versions will stay in sync

### CODE REVIEW

**Review by nastra — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -177,6 +177,8 @@
         <dep.casandra.version>4.17.0</dep.casandra.version>
         <dep.minio.version>8.5.6</dep.minio.version>
         <dep.iceberg.version>1.3.1</dep.iceberg.version>
+        <!-- Nessie version (matching to Iceberg release) must be bumped along with Iceberg version bump to avoid compatibility issues -->
+        <dep.nessie.version>0.59.0</dep.nessie.version>
```
Comment (findepi) [inline pom.xml:181]: This is needed by Iceberg only, so let's keep it in trino-iceberg/pom.xml

Comment (wendigo) [reply pom.xml:181]: This is just below iceberg version so seems consistent

Comment (wendigo) [reply pom.xml:181]: Ping @findepi

Comment (findepi) [reply pom.xml:181]: it still can be used in properties file, just not in `trino-testing-services`.

Comment (wendigo) [reply pom.xml:181]: Which will make this change more complicated

**Pull request closed without merging**